mergify_engine/actions/merge/action.py

```diff
@@ -93,31 +93,13 @@ def _required_statuses_in_progress(ctxt, missing_conditions):
         if ctxt.pull["state"] == "closed":
             return False
 
-        need_look_at_checks = []
         for condition in missing_conditions:
             if condition.attribute_name.startswith("status-"):
-                need_look_at_checks.append(condition)
+                # A CI is missing we need to wait
+                return True
             else:
                 # something else does not match anymore
                 return False
```

Comment:
So this is
```python
return condition.attribute_name.startswith("status-")
```
?
This is just using the first missing condition as the answer?

Reply:
yup

Reply:
I finally remove this commit, I don't get why yet but it doesn't work, so